recipe/build.sh

```diff
@@ -10,5 +10,14 @@ for i in `ls`; do
     [[ $i == "build_env_setup.sh" ]] && continue
     [[ $i == "conda_build.sh" ]] && continue
     [[ $i == "metadata_conda_debug.yaml" ]] && continue
-    cp -r $i ${PREFIX}
+    cp -rv $i ${PREFIX}
+
+    if [[ $i == "compute-sanitizer" ]]; then
+        for j in `ls $i`; do
+            [[ -d $PREFIX/$i/$j ]] && continue
+            [[ $j == "compute-sanitizer" ]] && continue
+
+            patchelf --set-rpath '$ORIGIN/../lib' $PREFIX/$i/$j
```

Comment:
Would it work if we used the same approach as we did to resolve issue ( https://github.com/conda-forge/cuda-feedstock/issues/10 )?

```suggestion
            patchelf --set-rpath '$ORIGIN' $PREFIX/$i/$j
```

Reply:
No, these will turn back to warnings:
```
INFO (cuda-sanitizer-api,compute-sanitizer/libTreeLauncherTargetInjection.so): Needed DSO lib/libgcc_s.so.1 found in conda-forge/linux-64::libgcc-ng==13.2.0=h807b86a_3
```

Reply:
Wonder if we could symlink `libgcc_s.so` into the same directory as `libTreeLauncherTargetInjection.so`

Maybe that would make the `patchelf` call unneeded

For more context, thinking about the `patchelf` call with `'$ORIGIN/../lib'`, am wondering if that will cause issues with the `targets` structure used in other packages (though admittedly not as relevant here)

Reply:
I don't see how managing that symlink could be at all preferable to what I have here.

Why are patchelf and the overlink check a concern here when we've already done that for every other cuda library package?

The libraries here are not installed to `targets`, or even `lib`, so you are right that it is not relevant here.